test-network-nano-bash/ca/createEnrollments.sh

```diff
@@ -0,0 +1,113 @@
+export peer1org2_dir=${org2_dir}/peers/peer1.org2.example.com
+
+export orderer_org_tls=${PWD}/data_ca/ordererca/ca/ca-cert.pem
+export org1_tls=${PWD}/data_ca/org1ca/ca/ca-cert.pem
+export org2_tls=${PWD}/data_ca/org2ca/ca/ca-cert.pem
+
+# import utilies
+. ca/ca_utils.sh
+
+######################################################################################
+#  Create admin certificates for the CAs
+######################################################################################
+
+# Enroll CA Admin for ordererca
+createEnrollment "7052" "admin" "adminpw" "" "${orderer_org_dir}/ca" "${orderer_org_tls}"
+
+# Enroll CA Admin for org1ca
+createEnrollment "7053" "admin" "adminpw" "org1" "${org1_dir}/ca" "${org1_tls}"
+
+# Enroll CA Admin for org2ca
+createEnrollment "7054" "admin" "adminpw" "org2" "${org2_dir}/ca" "${org2_tls}"
+
+
+######################################################################################
+#  Create admin certificates for the Organizations
+######################################################################################
+
+# Enroll Admin certificate for the ordering service org
+registerAndEnroll "7052" "osadmin" "osadminpw" "admin" "" "${orderer_org_dir}/users/Admin@example.com" "${orderer_org_dir}" "${orderer_org_tls}"
+
+# Enroll Admin certificate for org1
+registerAndEnroll "7053" "org1admin" "org1adminpw" "admin" "org1" "${org1_dir}/users/Admin@org1.example.com" "${org1_dir}" "${org1_tls}"
+
+# Enroll Admin certificate for org2
+registerAndEnroll "7054" "org2admin" "org2adminpw" "admin" "org2" "${org2_dir}/users/Admin@org2.example.com" "${org2_dir}" "${org2_tls}"
+
+
+######################################################################################
+#  Create the certificates for the Ordering Organization
+######################################################################################
+
+# Create enrollment and TLS certificates for orderer1
+registerAndEnroll "7052" "orderer1" "orderer1pw" "orderer" "" "${orderer1_dir}" "${orderer_org_dir}" "${orderer_org_tls}"
+
+# Create enrollment and TLS certificates for orderer2
+registerAndEnroll "7052" "orderer2" "orderer2pw" "orderer" "" "${orderer2_dir}" "${orderer_org_dir}" "${orderer_org_tls}"
+
+# Create enrollment and TLS certificates for orderer3
+registerAndEnroll "7052" "orderer3" "orderer3pw" "orderer" "" "${orderer3_dir}" "${orderer_org_dir}" "${orderer_org_tls}"
+
+# Create enrollment and TLS certificates for orderer4
+registerAndEnroll "7052" "orderer4" "orderer4pw" "orderer" "" "${orderer4_dir}" "${orderer_org}" "${orderer_org_tls}"
+
+# Create enrollment and TLS certificates for orderer5
+registerAndEnroll "7052" "orderer5" "orderer5pw" "orderer" "" "${orderer5_dir}" "${orderer_org_dir}" "${orderer_org_tls}"
+
+
+######################################################################################
+#  Create the certificates for Org1
+######################################################################################
```

Comment:
For simplicity nano doesn't switch between "admin" and "client" users, it just uses the "admin" user all the time.
However, since this is an educational sample, it is probably worthwhile to create the "client" users for Org1 and Org2.

Also, it may be easier to understand if the "admin" and "client" enrollments are both done here in the respective org section, especially since the title of this section is "Create the certificates for Org1", which for educational purposes should include all the org certs - admin, client, and peers.

Reply:
Added client enrollments to the script.